Shoko.Server/Mappings/AniDB_MessageMap.cs

```diff
@@ -0,0 +1,24 @@
+using FluentNHibernate.Mapping;
+using Shoko.Server.Models;
+using Shoko.Server.Server;
+
+namespace Shoko.Server.Mappings;
+
+public class AniDB_MessageMap : ClassMap<AniDB_Message>
+{
+    public AniDB_MessageMap()
+    {
+        Table("AniDB_Message");
+        Not.LazyLoad();
+        Id(x => x.AniDB_MessageID);
+
+        Map(x => x.MessageID).Not.Nullable();
+        Map(x => x.FromUserId).Not.Nullable();
+        Map(x => x.FromUserName).Not.Nullable();
+        Map(x => x.Date).Not.Nullable();
```

Comment:
Please rename the field to be less generic (E.g. "SentAt", "ReceivedAt", etc.). Also please also add a date for when it was fetched by Shoko ("E.g. "FetchedAt" or "CreatedAt", etc.).